specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-05-04/notebook.json

```diff
@@ -0,0 +1,461 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Cosmos DB",
+    "description": "Azure Cosmos DB Database Service Resource Provider REST API",
+    "version": "2020-05-04"
```

Comment:
I believe this is version "2020-04-01" in our RP? This is how the generated SDKs get API version, so they will see "api version does not exist"